src/core/client/stream/tabs/Comments/Stream/SortMenu.tsx

```diff
@@ -26,48 +28,55 @@ interface Props {
   reactionSortLabel: string;
 }
 
-const SortMenu: FunctionComponent<Props> = props => (
-  <MatchMedia ltWidth="sm">
-    {matches => (
-      <Flex
-        className={cn(props.className, CLASSES.sortMenu)}
-        justifyContent="flex-end"
-        alignItems="center"
-        itemGutter
-      >
-        {!matches && (
-          <Localized id="comments-sortMenu-sortBy">
-            <Typography
-              variant="bodyCopyBold"
-              container={<label htmlFor="coral-comments-sortMenu" />}
-            >
-              Sort By
-            </Typography>
-          </Localized>
-        )}
-        <SelectField
-          id="coral-comments-sortMenu"
-          value={props.orderBy}
-          onChange={props.onChange}
-          afterWrapper={(matches && <Icon>sort</Icon>) || undefined}
-          classes={{
-            select: (matches && styles.mobileSelect) || undefined,
-            afterWrapper: (matches && styles.mobileAfterWrapper) || undefined,
-          }}
+const SortMenu: FunctionComponent<Props> = props => {
+  const emitOpenSortMenuEvent = useViewerEvent(OpenSortMenuEvent);
+  const onClickSelectField = useCallback(() => emitOpenSortMenuEvent(), [
```

Comment:
Isn't this equivalent to just calling `emitOpenSortMenuEvent`? It looks like it's being wrapped in a `useCallback` just to rename the function.